azurerm/internal/services/sql/resource_arm_sql_server.go

```diff
@@ -100,6 +101,36 @@ func resourceArmSqlServer() *schema.Resource {
 				},
 			},
 
+			"blob_extended_auditing_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"storage_account_access_key": {
+							Type:         schema.TypeString,
+							Required:     true,
+							Sensitive:    true,
+							ValidateFunc: validate.NoEmptyStrings,
+						},
+						"storage_endpoint": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: validate.URLIsHTTPS,
+						},
+						"is_storage_secondary_key_in_use": {
```

Comment:
Is this a toggle for an option? i think it might make more sense as
```suggestion
						"storage_secondary_key_enabled": {
```

Its not entirely clear what this property does from the name